src/core/lattice/LatticeActionFactory.js

```diff
@@ -11,6 +11,15 @@ const acquireSyncTicket :RequestSequence = newRequestSequence(ACQUIRE_SYNC_TICKE
 const CREATE_ENTITY_AND_ASSOCIATION_DATA :'CREATE_ENTITY_AND_ASSOCIATION_DATA' = 'CREATE_ENTITY_AND_ASSOCIATION_DATA';
 const createEntityAndAssociationData :RequestSequence = newRequestSequence(CREATE_ENTITY_AND_ASSOCIATION_DATA);
 
+const FETCH_APP :'FETCH_APP' = 'FETCH_APP';
```

Comment:
I created a new library `lattice-sagas` to help reduce boilerplate and provide reusable sagas. The new actions you're adding, `FETCH_APP`, `FETCH_APP_CONFIGS`, `FETCH_APP_TYPES `, are exactly what `lattice-sagas` is all about.

Can you please move these actions as well as the corresponding sagas over to `lattice-sagas`? It'll basically be a direct copy/paste.